client/src/components/pages/RobotOverview/RobotOverview.jsx

```diff
@@ -115,6 +115,7 @@ const RobotOverview = () => {
             // eslint-disable-next-line no-underscore-dangle
             robotId={val._id}
             robotName={val.robotName}
+            refreshOverview={() => retrieveBotList(userId)}
```

Comment:
Just for my understanding, every time the Overview is reloaded after deleting a robot, a call is sent to the backend before that, right? Wouldn't it be easier to just change the local state of the Robot List? That would save us a backend call.